frontend/package.json

```diff
@@ -40,6 +40,7 @@
 		"react": "^17.0.2",
 		"react-dom": "^17.0.2",
 		"styled-components": "^5.3.0",
+		"uuid-validate": "^0.0.3",
```

Comment:
Would prefer to use own function instead of relying on like unmaintained package.

Reply:
```
function checkIfValidUUID(str) {
  // Regular expression to check if string is a valid UUID
  const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;

  return regexExp.test(str);
}
```
can?

Reply:
I'm not sure. This is the kind of thing that should have a unit test.